Test/myTest2

```diff
@@ -0,0 +1,20 @@
+from testing_base import *
+
+def aceTest():
+    one = ('Spade', 8)
+    two = ('ace')
+    
+    num_aces=0
+    non_ace_total=0
+    if num_aces > 0:
```

Comment:
Great work Jayden! But like Chris mentioned, this might become problematic if Ace is a variable throughout the game.

Reply:
Thanks! Yeah for sure, I would want to see how I could work my code with everyone else's for sure.